userlist.php

```diff
@@ -1,148 +1,14 @@
-<table>
```

Comment:
This file is redundant as I can have the code for it in settings.php. The new version is shortened.